tests/unit/test_credentials.py

```diff
@@ -96,14 +96,15 @@ async def test_env_credentials(monkeypatch, http):
     assert key.token == "token"
 
 
-async def test_ec2_instance_metdata_credentials(http, instance_metadata_server):
+@pytest.mark.parametrize("role", ["role", "arn:aws:iam::1234567890:role/test-role"])
```

Comment:
Does the new test only check that "client doesn't crash"?

Reply:
yes